deploy/workloads/bare-base.json

```diff
@@ -0,0 +1,12 @@
+{
+  "benchmark_name": "bare-base",
+  "common_bootbinary": "../../../../../software/firemarshal/images/firechip/bare-base/bare-base-bin",
+  "common_rootfs": "../../../../../software/firemarshal/images/firechip/bare-base/bare-base.img",
+  "common_outputs": [
+    "/etc/os-release"
+  ],
```

Comment:
Since this isn't Linux let's remove this.

Reply:
Done!